API/Osu/OsuEnums.cs

```diff
@@ -39,6 +39,9 @@ public enum Team
         Red = 2
     }
 
+    /// <summary>
+    /// Represents mod values
+    /// </summary>
     /// <copyright>
     /// ppy 2023 https://github.com/ppy/osu-api/wiki#mods
```

Comment:
Might also be worth adding documentation for when we got these values from this source (i.e. accessed April 2024). Just in case something changes we know where to go back in time to look.

```suggestion
    /// ppy 2024 https://github.com/ppy/osu-api/wiki#mods - Last referenced April 2024
```